browser-test/src/civiform_admin_question_list.test.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  AdminPrograms,
+  AdminQuestions,
+  createTestContext,
+  loginAsAdmin,
+} from './support'
+
+describe('Most recently updated question is at top of list.', () => {
```

Comment:
Note to the reviewer: This test was modeled off of the corresponding admin_program_list.test.ts (https://github.com/civiform/civiform/blob/08868c0bc54327e3ed4289c9827ba5c471f109e8/browser-test/src/admin_program_list.test.ts)